roadmap/self-hosting/README.md

```diff
@@ -67,6 +69,8 @@ Every update has a build number, and updates are only allowed to increase the nu
 
 A new release branch starts out "beta" but is marked "stable" at such a point as we feel it has proven itself in the wild.
 
+Admins should be able to change their update settings if they wish from the graphical user interface, without a fresh install.
```

Comment:
Just as a note, while there's no way to configure the sandstorm.conf file from the GUI (it wouldn't make sense for many of the options in that file, as they control the ability to see/access the GUI), you don't need a fresh install to change it. Just a service restart after changing the line in the config file. I'd like to see auto-update settings make it into the GUI at some point, though right now we don't have different branches anyways, so it's mostly an on/off thing.